app/views/manageBusinesses/manage/ManageIncomeSourceDetails.scala.html

```diff
@@ -137,6 +138,58 @@
     }
 }
 
+
+@getMTDUsageRows = @{
+
+    def getActionRow(useMTD: Boolean, taxYear: String, linkId: String) = {
+        val taxYearFormatted = s"${taxYear.toInt - 1}-$taxYear"
+        val changeTo = if (useMTD) "annual" else "quarterly"
+        val actionTextKey =
+            if (useMTD)
+                "incomeSources.manage.business-manage-details.OptInOptOutContentUpdateR17.optOut"
+            else
+                "incomeSources.manage.business-manage-details.OptInOptOutContentUpdateR17.signup"
+
+        val changeLink =
+            if (isAgent)
+                controllers.manageBusinesses.manage.routes.ConfirmReportingMethodSharedController
+                .showAgent(taxYearFormatted, changeTo, viewModel.incomeSourceType).url
+            else
+                controllers.manageBusinesses.manage.routes.ConfirmReportingMethodSharedController
+                .show(taxYearFormatted, changeTo, viewModel.incomeSourceType).url
+
+        SummaryListRow(
+            key = Key(Text(
+                messages(
+                    "incomeSources.manage.business-manage-details.OptInOptOutContentUpdateR17.mtdUsage",
+                    (taxYear.toInt - 1).toString,
+                    taxYear
+                )
+            )),
+            value = Value(Text(
+                if (useMTD)
+                    messages("incomeSources.manage.business-manage-details.OptInOptOutContentUpdateR17.mtdUsage.yes")
+                else
+                    messages("incomeSources.manage.business-manage-details.OptInOptOutContentUpdateR17.mtdUsage.no")
+            )),
+            actions = Some(Actions(
+                items = Seq(
+                    ActionItem(
+                        href = changeLink,
+                        content = Text(messages(actionTextKey)),
+                        attributes = Map("id" -> linkId)
+                    )
+                )
+            ))
+        )
+    }
+
+    Seq(
+        viewModel.useMTDForTaxYear1.map(getActionRow(_, viewModel.latencyDetails.get.taxYear1, "sign-up-link-1")),
+        viewModel.useMTDForTaxYear2.map(getActionRow(_, viewModel.latencyDetails.get.taxYear2, "sign-up-link-2"))
```

Comment:
Heya Ioannis 👋🏼 wondering please could we make these link ids dynamic for to match if it's a signup link or an opt out link? Just to help differentiate the them in ATs.  Perhaps 
opt-out-link-1 and opt-out-link-2?